packages/node_modules/@webex/react-container-read-receipts/src/selectors.js

```diff
@@ -1,7 +1,33 @@
 import {createSelector} from 'reselect';
 import {orderBy} from 'lodash';
+import {OrderedMap} from 'immutable';
 
-const getActivities = (state) => state.conversation.get('activities');
+const getConversationThreadActivities = (state) => state.conversation.get('threadActivities');
+const getConversationNonThreadActivities = (state) => state.conversation.get('sortNonThreadActivities');
+const getActivities = createSelector(
+  [
+    getConversationThreadActivities,
+    getConversationNonThreadActivities
+  ],
+  (
+    threadActivities = {},
+    sortNonThreadActivities = []
+  ) => {
+    const finalList = [];
+    let start = 0;
+
+    for (let i = 0; i < sortNonThreadActivities.length; i += 1) {
```

Comment:
Is it worthwhile to put this into a shared function somewhere instead of duplicating it?